helm/hpcc/templates/sasha.yaml

```diff
@@ -2,12 +2,14 @@
 {{- if not $sashaServices.disabled -}}
 {{ range $sashaName, $_sasha := $sashaServices -}}
 {{ $sasha := ($_sasha | default dict) -}}
-{{ if not $sasha.disabled -}}
-{{ $_ := set $sasha "name" $sashaName -}}
-{{ $_ := set $sasha "access" (splitList " " (include "hpcc.getSashaServiceAccess" $sasha)) -}}
-{{ $commonCtx := dict "root" $ "me" $sasha -}}
-{{ $configSHA := include "hpcc.sashaConfigMap" $commonCtx | sha256sum -}}
-{{ $serviceName := printf "sasha-%s" $sashaName -}}
+{{- if not $sasha.disabled -}}
+{{- $_ := set $sasha "name" $sashaName -}}
+{{- $_ := set $sasha "access" (splitList " " (include "hpcc.getSashaServiceAccess" $sasha)) -}}
+
```

Comment:
trivial: extra new line introduced?

Reply:
I've amended the last commit, removing the extra line.